src/main/java/kr/syeyoung/dungeonsguide/dungeon/mechanics/DungeonFairySoul.java

```diff
@@ -38,31 +39,16 @@ public class DungeonFairySoul implements DungeonMechanic {
     private List<String> preRequisite = new ArrayList<String>();
 
 
+    /**
```

Comment:
Although it's the same, It is completely different, and DungeonNPC does not fit this. I'd rather have repeating code here.